src/main/java/frc/robot/subsystems/CraneSubsystem.java

```diff
@@ -33,8 +57,63 @@ public void ShoulderDown() {
     craneSolenoids.set(Value.kReverse);
   }
 
-  public void MoveElbow(double elbow) {
-    elbowMotor.set(elbow);
+  public void DriveElbow() {
+    double position = absoluteEncoder.get();
+    double output = elbowPidController.calculate(position, elbowAngle);
+    if (output < -1) {
+      output = -1;
+    } else if (output > 1) {
+      output = 1;
+    }
+    elbowMotor.set(output);
+  }
+
+  public double getEncoderOutput() {
+    return absoluteEncoder.get();
```

Comment:
use get AbsolutePosition

Reply:
updated to consolidate to one function, it will use get in simulation and getAbsolute in reality